lib/xboxkrnl/xboxkrnl.h

```diff
@@ -3725,6 +3743,10 @@ XBAPI NTSTATUS NTAPI KeDelayExecutionThread
     IN PLARGE_INTEGER Interval
 );
 
+/**
+ * Connects an interrupt object, allowing it to receive interrupts
+ * @return FALSE if the interrupt is already connected or cannot be connected, TRUE if it was conected successfully.
```

Comment:
nit: This is the only parameter comment which ends in a period.

Reply:
Same as above, changed.